examples/acquiring_data/fido_metadata_queries.py

```diff
@@ -0,0 +1,53 @@
+"""
+====================================
+Querying Metadata clients using Fido
+====================================
+
+This example shows how to search and retrieve metadata using `~sunpy.net.Fido` from the
+search facilities like `~sunpy.net.hek.HEKClient`, `~sunpy.net.helio.HECClient`,
+and `~sunpy.net.jsoc.JSOCClient`. It also shows how to display desired columns in the result.
+"""
+from sunpy.net import Fido
+from sunpy.net import attrs as a
+
+###################################################################
+# We will query Helio for the 'rhessi_flare_list' table and
+# shall limit the total number of records to 5.
+# For the same time range, we will query HEK for 'FL' as the
+# Event Type and 'PeakFlux' greater than 1000.
+# We will also search JSOC for 'hmi.m_45s' Series.
+timerange = a.Time('2010/8/1 03:40', '2010/8/1 3:40:10')
+results = Fido.search(timerange, a.helio.TableName('rhessi_hxr_flare') |
+                      a.hek.FL & (a.hek.FL.PeakFlux > 1000) |
+                      a.jsoc.Series('hmi.m_45s') & a.jsoc.Notify("sunpy@sunpy.org"))
+print(results)
+
+###################################################################
+# Now we will download the searched records. Since HEK and HELIO
+# clients don't provide files, `Fido.fetch` will
+# ignore them and only download files from JSOC.
+files = Fido.fetch(results)
+print(files)
+
+###################################################################
+# Now we will extract individual responses from Fido results.
+# We can use ``get_response()`` to specify the name of the desired
```

Comment:
The more I look at `get_response`, it just seems too long and mis-named. We should come up with something better.

Reply:
Changed to `results['hek']` to get all hek responses from fido. If more than one results are from hek, then it returns a `UnifiedResponse`. If only one, then `HEKResponse` is returned. This is similar in all other clients.
Also, users can do `results['hek',1]` to get second `HEKResponse` and so on.

Reply:
Instead of `results['hek'][1]`?

Reply:
We can also do `results['hek'][1]` both will work.

Reply:
I am not sure I am comfortable with `results['hek'][1]` being the same as `results['hek',1]`